lib/v2/eastmoney/report/index.js

```diff
@@ -38,11 +38,15 @@ module.exports = async (ctx) => {
     const items = await Promise.all(
         list.map((item) =>
             ctx.cache.tryGet(item.link, async () => {
-                const { data: response } = await got(item.link);
-                const $ = cheerio.load(response);
-                item.link = $('.pdf-link').attr('href');
-                item.description = $('.ctx-content').text();
-                return item;
+                try {
+                    const { data: response } = await got(item.link);
+                    const $ = cheerio.load(response);
+                    item.link = $('.pdf-link').attr('href');
+                    item.description = $('.ctx-content').text();
+                    return item;
+                } catch (error) {
+                    return Promise.resolve('');
```

Comment:
```suggestion
                    return item;
```
You can still return part of the data from L31.

Reply:
ok